### ISSUE TITLE
See if we can speed things up with ancpBIDS

### ISSUE THREAD

**Issue by surchs** [neurobagel/bagel-cli#8]
Right now, parsing a large dataset takes an annoyingly long time. ancpBIDS is known to be faster. It should also support the core queries that we are interested in. So maybe we can remove the pyBIDS dependency, use ancpBIDS instead, and speed things up

- [ ] take a look to see if we could use ancpBIDS instead of pyBIDS
- [ ] migrate to ancpBIDS
- [ ] make a speed comparison for a large dataset and then decide what to keep

**Comment by surchs** [comment]
I would close this issue. It makes sense, but I don't think we'll get to it anytime soon. What do you think @alyssadai? If you agree, feel free to just close

**Comment by alyssadai** [comment]
@surchs I'm not super familiar with ancpBIDS, but I wonder if it'd be worth leaving this issue open at least until we've attempted to run the CLI on the PPMI and QPN datasets (which I think are a couple hundred subjects each)? I just did a quick search of "ancpBIDS" in the PyBIDS issues and based on this [post/comment](https://github.com/bids-standard/pybids/issues/831#issuecomment-1115370663) I suspect the long runtime might become an issue sooner than we think - it looks like it can take almost 6 min to validate a dataset of ~500 subjects. We can always just wait it out if that's the case, but I think at that point this issue might be worth revisiting, if only to figure out a game plan for the migration. Thoughts?
